mapping/core/src/main/java/it/unibz/inf/ontop/spec/fact/impl/FactExtractorWithSaturatedTBox.java

```diff
@@ -297,6 +297,7 @@ else if (e instanceof DataSomeValuesFrom) {
                 .filter(m -> m instanceof ObjectSomeValuesFrom)
                 .map(m -> (ObjectSomeValuesFrom) m)
                 .filter(m -> !m.getProperty().isInverse()) //Drop the inverse of itself generated by Ontop
+                .filter(m -> !m.getProperty().toString().contains("urn:AUX.ROLE")) // Drop auxiliary object property
```

Comment:
Test was failing with NPE (null object in a triple) presumably because the newer OWL API processes things in a different manner. I saw that urn:AUX.ROLE is filtered elsewhere in a similar manner.